### DISCUSSION TITLE
Uploading files with API

### DISCUSSION THREAD

**Discussion by kmkdz335** [drakkan/sftpgo#623] [Q&A] [answered]
Hello, drakkan! Im trying to upload files on sftp server using /api/v2/user/files, but on every request it gives me 404 response "not found". Im doing smth wrong or this endpoint is still in development?

**Comment by drakkan** [comment, marked as answer]
Hi,

yes, you have to use the development version. It will be v2.2.0 soon, so it's a good time to test it and report any issues you encounter, thank you

**Reply by kmkdz335** [reply]
It would be great if you can tell me even approximately date then it will be ready. Thank you for answer!

**Reply by drakkan** [reply]
No definite schedule sorry, it can happen tomorrow or next month, it depends on my free time. If you want to speed up the process consider a sponsorship so I can spend more time to the project, thanks.

However, the development version should be stable enough, I have a few things to fix but it should be good enough for standard use
